examples/jsm/postprocessing/OutlinePass.js

```diff
@@ -96,16 +96,14 @@ class OutlinePass extends Pass {
 		const copyShader = CopyShader;
 
 		this.copyUniforms = UniformsUtils.clone( copyShader.uniforms );
-		this.copyUniforms[ 'opacity' ].value = 1.0;
 
 		this.materialCopy = new ShaderMaterial( {
 			uniforms: this.copyUniforms,
 			vertexShader: copyShader.vertexShader,
 			fragmentShader: copyShader.fragmentShader,
 			blending: NoBlending,
 			depthTest: false,
-			depthWrite: false,
-			transparent: true
```

Comment:
The material already sets `blending` to `NoBlending`. Since the material is used in context of `FullScreenQuad`, we don't need the flag to force the quad into the transparent render list.